CitationStyleLanguagePlugin.php

```diff
@@ -426,14 +426,14 @@ public function getCitation(PKPRequest $request, Submission $submission, string
             $citationData->type = ($this->application === 'ojs2' ? 'article-journal' : 'article');
             $citationData->risType = 'JOUR';
             $citationData->id = $submission->getId();
-            $citationData->title = $publication->getLocalizedFullTitle();
+            $citationData->title = htmlspecialchars_decode($publication->getLocalizedFullTitle());
             $citationData->{'container-title'} = $context->getLocalizedName();
             $issueId = $publication->getData('issueId');
             $issue ??= $issueId ? Repo::issue()->get($issueId) : null;
             if ($issue) {
-                $citationData->volume = htmlspecialchars($issue->getData('volume'));
+                $citationData->volume = $issue->getData('volume');
                 // Zotero prefers issue and Mendeley uses `number` to store revisions
-                $citationData->issue = htmlspecialchars($issue->getData('number'));
+                $citationData->issue = $issue->getData('number');
```

Comment:
As the number can contain text, I think it's better to escape.

Reply:
But if the underlying library escapes data by itself, then there's nothing to worry about.